metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/utils/InputValue.java

```diff
@@ -14,7 +14,7 @@
  *  See the Licence for the specific language governing permissions and limitations under
  *  the Licence.
  */
```

Comment:
This class will be part of the API of the library. If we want to be extra safe, we could make this class unmodifiable (give it a constructor, remove all setters and copy/clone all arrays). What do you think?

Reply:
This should be part of our larger cleanup strategy.